### ISSUE TITLE
1.0.15 fails to compile (different error than 1.0.14)

### ISSUE THREAD

**Issue by hartwork** [svn-all-fast-export/svn2git#74]
Hi! I'm getting compile error with 1.0.15 that I'm not getting with 1.0.13. Any ideas what's wrong?

# Environment
- GCC 8.2.0
- Subversion 1.12.0
- Qt 5.12.3

# Compiler invocation
```
x86_64-pc-linux-gnu-g++ -c -march=core2 -msse4.1 --param l1-cache-line-size=64 --param l1-cache-size=32 --param l2-cache-size=6144 -O2 -pipe -Wall -W -D_REENTRANT -fPIC -DVER="\"1.0.15\"" -DQT_NO_DEBUG -DQT_CORE_LIB -I. -I. -isystem /usr/include/subversion-1 -isystem /usr/include/apr-1 -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtCore -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o repository.o repository.cpp
```
# Start of error output
```
repository.cpp: In member function ‘virtual void FastImportRepository::restoreBranchNotes()’:
repository.cpp:471:34: error: variable ‘QDataStream branchNotesStream’ has initializer but incomplete type
     QDataStream branchNotesStream(&branchNotesFile);
                                  ^
repository.cpp: In member function ‘virtual void FastImportRepository::saveBranchNotes()’:
repository.cpp:798:34: error: variable ‘QDataStream branchNotesStream’ has initializer but incomplete type
     QDataStream branchNotesStream(&branchNotesFile);
                                  ^
```
# Full build log
(From trying to update the Gentoo package of it, hence the color and Gentoo bits)
[svn2git-1.0.15-build-log.txt](https://github.com/svn-all-fast-export/svn2git/files/3134752/svn2git-1.0.15-build-log.txt)

# Related
- issue #73

**Comment by gabyx** [comment]
Have the same issue.

**Comment by hartwork** [comment]
Seems to be fixed in 1.0.16, closing.
